versioning_base.py

```diff
@@ -213,14 +213,24 @@ def checkout(pg_conn_info, pg_table_names, sqlite_filename):
         # use ogr2ogr to create spatialite db
         if first_table:
             first_table = False
-            cmd = ['ogr2ogr',
-                '-preserve_fid',
-                '-f', 'SQLite',
-                '-dsco', 'SPATIALITE=yes',
-                sqlite_filename,
-                'PG:"'+pg_conn_info+'"', schema+'.'+table,
-                '-nln', table]
-            #print ' '.join(cmd)
+            if feature_list:
+                cmd = ['ogr2ogr',
+                    '-preserve_fid',
+                    '-f', 'SQLite',
+                    '-dsco', 'SPATIALITE=yes',
+                    sqlite_filename,
+                    'PG:"'+pg_conn_info+'"', schema+'.'+table,
+                    '-nln', table,
+                    '-where', '"'+pkey+' in ('+",".join([str(feature_list[i][pkey]) for i in range(0, len(feature_list))])+')"']
+            else:
+                cmd = ['ogr2ogr',
+                    '-preserve_fid',
+                    '-f', 'SQLite',
+                    '-dsco', 'SPATIALITE=yes',
+                    sqlite_filename,
+                    'PG:"'+pg_conn_info+'"', schema+'.'+table,
+                    '-nln', table]
+            print ' '.join(cmd)
```

Comment:
Since the feature list only adds the -where option, I'd rather have a something like:
```python
cmd = ['ogr2ogr',
                    '-preserve_fid',
                    '-f', 'SQLite',
                    '-dsco', 'SPATIALITE=yes',
                    sqlite_filename,
                    'PG:"'+pg_conn_info+'"', schema+'.'+table,
                    '-nln', table] \
                    + ['-where', '"'+pkey+' in ('+",".join([str(feature_list[i][pkey]) for i in range(0, len(feature_list))])+')"'] \
                        if feature_list else []
```